server/api/core/core.js

```diff
@@ -162,6 +174,11 @@ export default {
         _id: 1
       }
     }).fetch()[0];
+
+    if (!shop) {
+      Logger.debug(domain, "Add a domain entry to shops for ");
```

Comment:
Agreed.  Definitely feels like a `warn` scenario.  And I think it'd be a good idea to clarify the output a little.  I usually make the first arg an object so it's a little clearer what you're seeing when that outputs to the console (or wherever).  So...

```js
Logger.warn({ domain }, "Shop not found");
```

Otherwise, when you use a string for that first arg, it renders that as if it's the main log message.

Or maybe better yet, just use a single arg as a string...

```js
Logger.warn(`Add a domain entry to shops for ${domain}`);
```

But I also think it could use a more helpful message for debugging.  Perhaps...

```js
Logger.warn(`getShopId: Shop not found with domain ${domain}`);
```

That tells you where and why it happened.